<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VenderProdutos</class>
 <widget class="QDialog" name="VenderProdutos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>865</width>
    <height>325</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="FPag_txt">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>241</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Forma Pagamento</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="End_txt">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>241</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Endere√ßo</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Data_txt">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>140</y>
     <width>241</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Data</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Pedido</string>
   </property>
  </widget>
  <widget class="QPushButton" name="FinalizaP_bt">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>260</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Finalizar Pedido</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>100</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Nome Completo</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Id_txt">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>57</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Nome_txt">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>100</y>
     <width>301</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>60</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Id Funcionario</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Quantidade</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Produto_txt">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>220</y>
     <width>241</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Qtd_txt">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>260</y>
     <width>241</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Produto</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AdProduto_bt">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>210</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Adcionar Produto</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
